packages/server/graphql/mutations/deleteComment.ts

```diff
@@ -50,7 +49,8 @@ const deleteComment = {
       return {error: {message: `Not a member of the meeting`}}
     }
     const {createdBy, discussionId} = comment
-    if (!isDiscussionFromMeeting(discussionId, meeting)) {
+    const discussion = await dataLoader.get('discussions').loadNonNull(discussionId)
+    if (discussion.meetingId !== meetingId) {
```

Comment:
`discussion.meetingId === meetingId` is probably the best and simplest source of truth for whether a discussion is part of the meeting, which is why I fixed this by simply fetching the discussion here instead of trying to implement `isDiscussionFromMeeting` for check-ins and standups